authenticator/authenticator.go

```diff
@@ -220,7 +220,7 @@ func computePGSHA256SaltedPass(password string, salt string, iterations int) ([]
 func xorBytes(a, b []byte) []byte {
 	buf := make([]byte, len(a))
 
-	for i, _ := range a {
+	for i := range a {
```

Comment:
Would it be possible to add a simple happy-path unit test for the `xorBytes` method? Then it helps guarantee that we're still getting what we'd expect out of it.

Reply:
Just added that!